components/camel-cxf/camel-cxf-common/src/main/java/org/apache/camel/component/cxf/common/header/CxfHeaderHelper.java

```diff
@@ -50,7 +50,6 @@ public final class CxfHeaderHelper {
         defineMapping(Exchange.CONTENT_TYPE, Message.CONTENT_TYPE);
         defineMapping(Exchange.HTTP_CHARACTER_ENCODING, Message.ENCODING);
         defineMapping(Exchange.HTTP_QUERY, Message.QUERY_STRING);
-        defineMapping(Exchange.ACCEPT_CONTENT_TYPE, Message.ACCEPT_CONTENT_TYPE);
```

Comment:
The ACCEPT_CONTENT_TYPE is never used internally. This method was a mapping, and in theory the user should have been already moved to CXF message accept content type instead.